### ISSUE TITLE
Using the holiday file configured in timewarrior.cfg

### ISSUE THREAD

**Issue by Blackjacx** [guludo/timewarrior-balance#8]
It looks like you only use the `en-US` holiday file to exclude public holidays. Is that possible? In Germany we have `Jan 1st` off and I explicitly have to enter it in `balance.conf`. This could be a hint that the extension does not take into account when a user specifies `import /opt/homebrew/share/doc/timew/holidays/holidays.de-DE` in timewarrior.cfg.

**Comment by guludo** [comment]
Actually, timewarrior-balance does not use any holiday file at the moment. We could think of a way of using it. Patches are welcome :-)

**Comment by Blackjacx** [comment]
Ah so that means each holiday in any country should be entered in balance.conf when I want to exclude it from calculations.

**Comment by guludo** [comment]
Yes. That's the current way to do it. I think using an external holiday file has some value, but we need to think of how that should be implemented: given a holiday, we need to make timewarrior-balance know what tags that would be applied to and how much time to consider. For the latter, maybe a sane default would be to match the hours of whatever config entry is on that day.
